charts/argo/templates/server-cluster-roles.yaml

```diff
@@ -29,11 +29,20 @@ rules:
   - list
   - watch
   - delete
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - watch
+  - create
+  - patch
 {{- if .Values.controller.persistence }}
 - apiGroups:
   - ""
   resources:
   - secrets
+  - serviceaccounts
```

Comment:
We are giving argo-server wide ranging permissions here, not in this PR but in the future we sdhould maybe support a whitelist for secrets and serviceaccounts so that it can load only the ones a user specifies and things are locked down a llittle.

Reply:
maybe you'd like to create an open issue for that?